llvm/lib/Target/SPIRV/SPIRVModuleAnalysis.cpp

```diff
@@ -630,6 +630,15 @@ void RequirementHandler::initAvailableCapabilities(const SPIRVSubtarget &ST) {
   addAvailableCaps({Capability::Shader, Capability::Linkage, Capability::Int8,
                     Capability::Int16});
 
+  if (ST.isAtLeastSPIRVVer(VersionTuple(1, 3)))
```

Comment:
I'm not sure why this needed to move?

Reply:
This needed to move so that we can use the target triple `mtriple=spirv-unknown-unknown` in the testcase. Previously we would get an `Unable to meet SPIR-V requirements for the target` error with `Capability not yet supported: GroupNonUniformVote`. This was incorrect as these capabilities are all supported as part of vulkan 1.1 -> spirv 1.3.